src/app/app.routing.ts

```diff
@@ -124,7 +125,12 @@ const APP_ROUTES: Routes = [
   { path: 'onboarding', component: OnboardingComponent, canActivate: [AuthGuard], data: { title: 'Dockstore | Onboarding' } },
   { path: 'accounts', component: AccountsComponent, canActivate: [AuthGuard], data: { title: 'Dockstore | Accounts' } },
   { path: 'auth/:provider', component: AuthComponent },
-  { path: 'starred', component: StarredEntriesComponent, data: { title: 'Dockstore | Starred Tools & Workflows' } },
+  {
+    path: 'starred',
+    component: StarredEntriesComponent,
+    canActivate: [AuthGuard],
+    data: { title: 'Dockstore | Starred Tools & Workflows' },
```

Comment:
It was like this, but I just noticed in reviewing this PR, that the page also shows starred organizations, so text is wrong.